### PR TITLE
Appsignal issue fix for contact

### PR DESCRIPTION
**Pull request by mdshamoon**
## Summary

Fixed no contact error.

### CODE REVIEW

**Comment by kurund** [comment]
@mdshamoon 

Let's try to handle the scenario where the user is trying to get the information about the contact i.e. is not in cache.

**Review by kurund — CHANGES_REQUESTED** [review]
@mdshamoon

Let's try to handle the scenario where the user is trying to get the information about the contact i.e. is not in cache.

File: src/containers/Chat/ChatMessages/ChatMessages.tsx [hunk 1]
```diff
@@ -221,6 +221,11 @@ export const ChatMessages: React.SFC<ChatMessagesProps> = ({
 
   // HOOKS ESTABLISHED ABOVE
 
+  // check if the search API results nothing for a particular contact ID and redirect to chat
+  if (contactId && data && data.search.length === 0) {
```
Comment (DigneshGujarathi) [inline src/containers/Chat/ChatMessages/ChatMessages.tsx:225]: Let's merge these two conditions into one.
if (contactId && data) {
  if (data.search.length === 0 || data.search[0].contact.status === 'BLOCKED') {	
    return <Redirect to="/chat" />;
  }
}

Comment (mdshamoon) [reply src/containers/Chat/ChatMessages/ChatMessages.tsx:225]: Sure

**Review by DigneshGujarathi — COMMENTED** [review]
Looks good

**Pull request merged**